scripts/app.js

```diff
@@ -44,7 +44,8 @@ H5P.ThreeImage = (function () {
       showCode: 'Show code',
       hideCode: 'Hide code',
       unlockedStateAction: 'Continue',
-      lockedStateAction: 'Unlock',
```

Comment:
?

Reply:
Must have disappeard by accident, readded it no